platform/ui/src/types/index.ts

```diff
@@ -15,3 +14,9 @@ const StringNumber = PropTypes.oneOfType([PropTypes.string, PropTypes.number]);
 const StringArray = PropTypes.oneOfType([PropTypes.string, PropTypes.array]);
 
 export { StringNumber, StringArray, ThumbnailType };
+
+export type IProps = Record<string, unknown>;
+
+export type Predicate = (props: IProps) => boolean;
+
+export type Object = Record<string, unknown>;
```

Comment:
I don't think any of these are needed. They give feeling of being typesafe, though providing no value at all regarding type

Reply:
Left predicate, but removed the others.  Predicate is a standard definition in many programming languages, so is useful to have.